apps/api/src/expenses/expenses.controller.ts

```diff
@@ -46,11 +62,60 @@ export class ExpensesController {
   }
 
   @Delete(':id')
+  remove(@Param('id') id: string) {
+    return this.expensesService.remove(id)
+  }
+
+  @Get('campaign/:slug')
   @Roles({
     roles: [RealmViewSupporters.role, ViewSupporters.role],
     mode: RoleMatchingMode.ANY,
   })
-  remove(@Param('id') id: string) {
-    return this.expensesService.remove(id)
+  async listCampaignExpenses(@Param('slug') slug: string) {
+    return this.expensesService.listCampaignExpenses(slug)
+  }
```

Comment:
in general I would move the listCampaignExpenses to the campaign controler - so that the full route is campaigns/:slug/expenses

Reply:
moved it.